index.html

```diff
@@ -1,3 +1,59 @@
-﻿
+<!doctype html>
+<html lang="en">
+<head>
+<meta name="description" content="Demo project">
+<title>Cookie Recipe</title>
+</head>
 
+<body>
+<header>
+  <h1>The Best Chocolate Chip Cookies </h1>
+  </header>
 
+<section>
+  <img src="https://raw.githubusercontent.com/Coderbabez/2019-spring-01h-recipe-page/master/img/cookies.jpg"
```

Comment:
2 things here: 

The browser is smart and figured out the right thing to do, but for completeness, you should close the image tag like: `<img src="..." />` with that closing `/>`

You can make it easier to include an image by using a relative path. Instead of listing the full url, you can state where the image file is relative to where this file is. like`<img src=`img/cookies.jpg' />`